qupulse/hardware/awgs/zihdawg_qi.py

```diff
@@ -556,14 +556,18 @@ def upload(self, name: str,
         #q_sample_rate = float(self.sample_rate/ 10**9)
 
         # Adjust program to fit criteria.
+        t_start = time.time()
```

Comment:
For tasks like this use `time.perf_counter()` instead of `time.time()` @mjwoudstra